plugins/linux-capture/pipewire.c

```diff
@@ -82,6 +89,11 @@ struct _obs_pipewire_data {
 
 	struct pw_core *core;
 	struct spa_hook core_listener;
+	int server_version_sync;
+
+	struct obs_pw_version server_version;
+	struct obs_pw_version library_version;
+	struct obs_pw_version header_version;
```

Comment:
Now that we explicitly depend on 0.3.40, I think we don't need `library_version` nor `header_version` - they're not used in this PR anyway